README.md

```diff
@@ -11,8 +11,10 @@
 ----
 
 Hazelcast is an open-source distributed in-memory data store and
```

Comment:
The first sentence in this paragraph seems long and complex.

Perhaps we can break it down. Something like:

> Hazelcast is a distributed in-memory data store and computation platform.

> As an in-memory data store, Hazelcast gives you faster access to your data by storing it in memory. With more accessible data, you can leverage Hazelcast to process huge amounts of real-time events or static datasets with consistently low latency.

> To help you take advantage of all these features, Hazelcast comes with the following built-in data structures: